pyquil/tests/test_operator_estimation.py

```diff
@@ -1758,7 +1915,7 @@ def test_corrected_statistics_2q_nontrivial_nonentangled_state(forest):
     expected_std_err = np.sqrt(np.var(expectations))
     # compare against simulated results
     np.testing.assert_allclose(result_expectation, expected_expectation, atol=2e-2)
-    np.testing.assert_allclose(result_std_err, expected_std_err, atol=2e-2)
+    np.testing.assert_allclose(result_std_err, expected_std_err, atol=3e-2)
```

Comment:
This seems unrelated to the PR description

Reply:
Reverted this back to original `atol` and picked appropriate seeds.